### ISSUE TITLE
ActiveRecord::AdapterTest#test_select_all_with_legacy_binds fails

### ISSUE THREAD

**Issue by yahonda** [rsim/oracle-enhanced#1246]
rails/rails#27939

#### Environments:
Rails 5.1.0.rc1
Oracle enhanced adapter 1.8.0.rc1

```ruby
$ ARCONN=oracle bundle exec ruby -w -Itest test/cases/adapter_test.rb -n test_select_all_with_legacy_binds
/home/yahonda/git/rails/activesupport/lib/active_support/core_ext/enumerable.rb:20: warning: method redefined; discarding old sum
Using oracle
Run options: -n test_select_all_with_legacy_binds --seed 50103

# Running:

E

Finished in 0.174951s, 5.7159 runs/s, 0.0000 assertions/s.

  1) Error:
ActiveRecord::AdapterTest#test_select_all_with_legacy_binds:
NoMethodError: undefined method `value_for_database' for [nil, 10000]:Array
    /home/yahonda/git/oracle-enhanced/lib/active_record/connection_adapters/oracle_enhanced/database_statements.rb:20:in `block in exec_query'
    /home/yahonda/git/oracle-enhanced/lib/active_record/connection_adapters/oracle_enhanced/database_statements.rb:20:in `map'
    /home/yahonda/git/oracle-enhanced/lib/active_record/connection_adapters/oracle_enhanced/database_statements.rb:20:in `exec_query'
    /home/yahonda/git/rails/activerecord/lib/active_record/connection_adapters/abstract/database_statements.rb:361:in `select'
    /home/yahonda/git/rails/activerecord/lib/active_record/connection_adapters/abstract/database_statements.rb:42:in `select_all'
    /home/yahonda/git/rails/activerecord/lib/active_record/connection_adapters/abstract/query_cache.rb:97:in `select_all'
    test/cases/adapter_test.rb:223:in `test_select_all_with_legacy_binds'

1 runs, 0 assertions, 0 failures, 1 errors, 0 skips
$
```

**Comment by koic** [comment]
#1248 fixes this issue.

**Comment by yahonda** [comment]
Closed by #1248
